dashboard/modules/job/tests/test_job_manager.py

```diff
@@ -18,6 +18,11 @@
 
 @pytest.fixture(scope="session")
 def shared_ray_instance():
+    # Remove ray address for test ray cluster in case we have
+    # lingering RAY_ADDRESS="http://127.0.0.1:8265" from previous local job
+    # submissions.
+    if "RAY_ADDRESS" in os.environ:
+        del os.environ["RAY_ADDRESS"]
```

Comment:
Nit: Should we save the old `RAY_ADDRESS` and reinstate it after the `yield`?  (Actually can't we automatically do this with `monkeypatch.delenv` or something similar?)

Reply:
i also tried monkeypatch but it didn't directly work with session level fixture however. This line is rarely necessary and completely irrelevant on CI, and the only case would be user is trying to run job_manager unit test but somehow manually set RAY_ADDRESS="http://127.0.0.1:8265" from previous attempts of job submission tutorial, like how you originally run into the issue.